octa/assetlist.json

```diff
@@ -0,0 +1,27 @@
+{
+  "chain_id": "octa",
+  "assets": [
+    {
+      "description": "The native token of Octa Chain",
+      "denom_units": [
+        {
+          "denom": "uocta",
+          "exponent": 0,
+          "aliases": []
+        },
+        {
+          "denom": "octa",
+          "exponent": 6,
+          "aliases": []
+        }
+      ],
+      "base": "uocta",
+      "display": "octa",
+      "symbol": "OCTA",
+      "logo_URIs": {
+        "png": "https://octa-coin-assets.s3.eu-central-1.amazonaws.com/octa-512x512.png",
```

Comment:
Please upload these images to this repo under octo/images/osct.png ,and .svg and point these URIs to that location

Reply:
please also change this URI to `"https://raw.githubusercontent.com/cosmos/chain-registry/master/octa/images/octa.png"`